### ISSUE TITLE
Automatically build a new docker image when a new ansible version is out

### ISSUE THREAD

**Issue by GabLeRoux** [GabLeRoux/ansible-docker-image#3]
This should be done after #2 
It would be very nice to have such a feature so I wouldn't have to commit and push a commit and keep this up to date with the upstream. We need set it and forget it solutions. This project deserves this.

There are build triggers in docker hub:

![image](https://user-images.githubusercontent.com/1264761/49701538-bb9ec200-fbbb-11e8-9e83-e7338ab5f0c9.png)

And there are webhooks: https://docs.docker.com/docker-hub/webhooks/

Related article:
https://medium.com/mobileforgood/patterns-for-continuous-integration-with-docker-on-travis-ci-71857fff14c5

Related tools (maybe?)

* https://requires.io/
* https://pyup.io/
* https://greenkeeper.io/

**Comment by GabLeRoux** [comment]
I've subscribed to requires.io pull-requests, should be fine like this for now

![image](https://user-images.githubusercontent.com/1264761/49702262-3cfa5280-fbc4-11e8-8881-143b665994bc.png)
